<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryParameters</class>
 <widget class="QMainWindow" name="QueryParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>590</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>682</width>
    <height>590</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Higiia - Query Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../icons.qrc">
    <normaloff>:/icons/favicon.ico</normaloff>:/icons/favicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background: black;
	color: white;
}

QFrame{
	border: none;
}

QWidget{
	color: white;
	font-size: 9pt;
}



QLabel{
	color: white;
}



QGroupBox {
     color: black;
     border: 1px solid gray;
     border-radius: 0px;
     margin-top: 1ex; /* leave space at the top for the title */
 }

 QGroupBox::title {
     color: black;
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 3px;
 }

 QToolBox::tab {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
     border-radius: 5px;
     color: black;
 }

 QToolBox::tab:selected { /* italicize selected tabs */
     font: italic;
 }

 QLineEdit {
     color: white;
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: white;
     selection-background-color: darkgray;
 }


 QPushButton {
     border: 2px solid #8f8f91;
     border-radius: 6px;
     min-width: 80px;
	color: white;
 }

 QPushButton:flat {
     border: none; /* no border for a flat push button */
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">	background: black;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="2" column="2">
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>64</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnClose">
         <property name="minimumSize">
          <size>
           <width>84</width>
           <height>82</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>82</width>
           <height>82</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../icons.qrc">
           <normaloff>:/icons/icoClose.png</normaloff>:/icons/icoClose.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPACS">
         <property name="minimumSize">
          <size>
           <width>84</width>
           <height>82</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>82</width>
           <height>82</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>21</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget{
	font-size: 21pt;
}</string>
         </property>
         <property name="text">
          <string>PACS</string>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnViewStudy">
         <property name="minimumSize">
          <size>
           <width>84</width>
           <height>82</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>82</width>
           <height>82</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../icons.qrc">
           <normaloff>:/icons/icoCbirSearch.png</normaloff>:/icons/icoCbirSearch.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QFrame" name="frame_4">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lblServerStatusConst">
         <property name="text">
          <string>App Status: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblServerSetup">
         <property name="text">
          <string>Connected to the Server!</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>335</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolBox" name="toolBoxSearchOptions">
            <property name="minimumSize">
             <size>
              <width>280</width>
              <height>385</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>280</width>
              <height>385</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <widget class="QWidget" name="pageQueryParameters">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>280</width>
               <height>323</height>
              </rect>
             </property>
             <property name="maximumSize">
              <size>
               <width>280</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <attribute name="label">
              <string>Search Parameters</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="lblHypothesis">
                <property name="text">
                 <string>Hypothesis:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cbxHypothesis">
                <property name="styleSheet">
                 <string notr="true">QWidget {
	color: white;
}
</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lblMinimum">
                <property name="text">
                 <string>Minimum value:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtMinimum"/>
              </item>
              <item>
               <widget class="QLabel" name="lblMaximum">
                <property name="text">
                 <string>Maximum value:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtMaximum"/>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="pageSimParameters">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>266</width>
               <height>325</height>
              </rect>
             </property>
             <property name="maximumSize">
              <size>
               <width>280</width>
               <height>16777215</height>
              </size>
             </property>
             <attribute name="label">
              <string>Similarity Parameters</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout">
              <item row="11" column="0" colspan="3">
               <widget class="QRadioButton" name="rbtBridge">
                <property name="text">
                 <string>Bridged Similarity Search</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0" colspan="2">
               <widget class="QLabel" name="lblDiversityNumber">
                <property name="text">
                 <string>Max group size:</string>
                </property>
               </widget>
              </item>
              <item row="6" column="2">
               <widget class="QLineEdit" name="txtNumberNeighbors">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>15</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0" colspan="3">
               <widget class="QComboBox" name="cbxDf">
                <property name="styleSheet">
                 <string notr="true">QWidget {
	color: white;
}
</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0" colspan="3">
               <widget class="QRadioButton" name="rbtSimilarity">
                <property name="text">
                 <string>Similarity Search</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="0" colspan="3">
               <widget class="QLabel" name="lblSimAttribute">
                <property name="text">
                 <string>Similarity Attribute:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0" colspan="2">
               <widget class="QLabel" name="lblDistanceEvaluation">
                <property name="text">
                 <string>Distance Function:</string>
                </property>
               </widget>
              </item>
              <item row="12" column="0">
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="4" column="0" colspan="3">
               <widget class="QLabel" name="lblRelevanceFeedbackMethod">
                <property name="text">
                 <string>Relevance Feedback:</string>
                </property>
               </widget>
              </item>
              <item row="7" column="2">
               <widget class="QLineEdit" name="txtNumberDiversity">
                <property name="text">
                 <string>10</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="lblNumberOfNeighbors">
                <property name="text">
                 <string>Neighbors:</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0" colspan="3">
               <widget class="QComboBox" name="cbxRF">
                <property name="styleSheet">
                 <string notr="true">QWidget {
	color: white;
}
</string>
                </property>
               </widget>
              </item>
              <item row="9" column="0" colspan="3">
               <widget class="QRadioButton" name="rbtDiversity">
                <property name="text">
                 <string>Diversity Search</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="3">
               <widget class="QComboBox" name="cbxSimilarityAttribute">
                <property name="styleSheet">
                 <string notr="true">QWidget {
	color: white;
}
</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lblImg">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>300</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>300</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer_8">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="2">
     <spacer name="horizontalSpacer_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="2">
     <spacer name="horizontalSpacer_11">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0">
     <spacer name="horizontalSpacer_9">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>17</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <spacer name="verticalSpacer_9">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="1">
     <spacer name="verticalSpacer_10">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>verticalSpacer_4</zorder>
   <zorder>frame_4</zorder>
   <zorder>verticalSpacer_2</zorder>
   <zorder>verticalSpacer_9</zorder>
   <zorder>horizontalSpacer_4</zorder>
   <zorder>horizontalSpacer_6</zorder>
   <zorder>horizontalSpacer_7</zorder>
   <zorder>horizontalSpacer_9</zorder>
   <zorder>horizontalSpacer_10</zorder>
   <zorder>horizontalSpacer_11</zorder>
   <zorder>verticalSpacer_10</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cbxHypothesis</tabstop>
  <tabstop>txtMinimum</tabstop>
  <tabstop>txtMaximum</tabstop>
  <tabstop>cbxSimilarityAttribute</tabstop>
  <tabstop>cbxDf</tabstop>
  <tabstop>cbxRF</tabstop>
  <tabstop>txtNumberNeighbors</tabstop>
  <tabstop>txtNumberDiversity</tabstop>
  <tabstop>rbtSimilarity</tabstop>
  <tabstop>rbtDiversity</tabstop>
  <tabstop>rbtBridge</tabstop>
  <tabstop>btnViewStudy</tabstop>
  <tabstop>btnPACS</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
